test/fail_compilation/checkimports3.d

```diff
@@ -0,0 +1,15 @@
+/*
+REQUIRED_ARGS: -transition=checkimports -de
+TEST_OUTPUT:
+---
+fail_compilation/checkimports3.d(14): Deprecation: local import search method found overloadset checkimports3.foo (3 overloads) instead of overloadset checkimports3.foo (2 overloads)
```

Comment:
> You can't really fix the deprecations, as long as you pass -transition=checkimports the compiler lists differences between the old and new lookup/import rules that must be inspected manually.

Hmm, seems weird that it's not possible fix a deprecation warning. If I cannot fix a deprecation warning how do I know that my code what cause issue in the future? I might have completely misunderstood this, but I thought enabling  `-transition=checkimports` would warn when the current code will not have the same behavior with the new lookup rules? Is the new lookup rules already in place by default?